cargo-shuttle/src/provisioner_server.rs

```diff
@@ -116,7 +116,11 @@ impl LocalProvisioner {
                     .expect("container to be created")
             }
             Err(error) => {
-                error!("got unexpected error while inspecting docker container: {error}");
+                error!(
+                    "got unexpected error while inspecting docker container: {error}. If you're \
+                    using Podman, view these instructions: \
+                    https://docs.rs/shuttle-runtime/latest/shuttle_runtime/#using-podman-instead-of-docker"
+                );
```

Comment:
I also think it could be worth figuring out which variant of the error is triggered by missing docker, then show your improved error message for that, and keep the "unexpected" error message as a catch all for the remaining variants. :thinking: